src/aura/Work_Items_List/Work_Items_ListController.js

```diff
@@ -1,4 +1,20 @@
 ({
-	myAction : function(component, event, helper) {
+	doInit : function(component, event, helper) {
+		var statusType = component.get('v.workItemsStatus');
+
+		var action = component.get('c.getWorkItemsByStatusInLast30Days');
+		action.setParam('status', statusType);
+
+		action.setCallback(this, function(response){
+			var state = response.getState();
+			if(state == 'SUCCESS'){
```

Comment:
Using strict equal and strict not equal instead of the other versions helps to maintain data type integrity through the code.